chart/values.yaml

```diff
@@ -475,9 +476,11 @@ loki-stack:
     persistence:
       # -- Enable persistence storage for the logs
       enabled: true
-      # -- StorageClass for Loki's centralised log storage
-      # empty storageClass implies cluster default storageClass & `manual` creates a static hostpath PV
-      storageClassName: ""
+      # -- StorageClass for Loki's centralised log storage. Options:
+      # <p> - `"mayastor-openebs-hostpath"` - Will provision openebs local-pv hostpath volume. Needs `Values.localpv-provisioner.enabled = true`. <br>
+      # - `"manual"` - Will provision a hostpath PV on the same node. <br>
+      # - `""` (empty) - Will use the default StorageClass on the cluster. </p>
+      storageClassName: "mayastor-openebs-hostpath"
```

Comment:
Should the name have localpv in it so that it makes it clear of the intention? Something like openebs-localpv-hostpath?

Reply:
Done.